File: jtaf-web/src/java/ch/jtaf/entity/SecurityGroup.java
package ch.jtaf.entity;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class SecurityGroup {

    @Id
    private String email;
    private String name;

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: src/main/kotlin/ch/jtaf/entity/SecurityGroup.kt
package ch.jtaf.entity

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType.IDENTITY
import javax.persistence.Id

@Entity
data class SecurityGroup(
        @Id @GeneratedValue(strategy = IDENTITY)
        var id: Long? = null,

        var name: String = ""
)